.gitignore

```diff
@@ -2,3 +2,4 @@ app/config/config.local.neon
 node_modules
 .tmp
 public/data/history/*
+/nbproject/private/
```

Comment:
prosim tyhle ignor radky do sveho lokalniho ignor file https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
OK